accelerator/roles/accelerator_validation/tasks/main.yml

```diff
@@ -17,8 +17,14 @@
   ansible.builtin.set_fact:
     control_plane_os: "{{ ansible_distribution | lower }}"
 
+- name: Saving distribution version of os
+  ansible.builtin.set_fact:
+    control_plane_os_version: "{{ ansible_distribution_version | lower }}"
+
 - name: Include local_repo variables
   ansible.builtin.include_tasks: include_local_repo_config.yml
 
+# TODO: need a way to differentiate platforms, then run different validations
 - name: Check xcat installation status
-  ansible.builtin.include_tasks: validate_amd.yml
```

Comment:
please do not remove this, instead add new task

Reply:
same as above